### PR TITLE
GPII-4008: Morphic 1.2.0 EARLY Candidate

### PR DESCRIPTION
**Pull request by krisYanachkov**
- G1 - Scroll problem still exists for language menu

- G3 - If muted - then hit vol up or down should “unmute” and take you back to previous volume

-  Added footer link to 'Zoom top window' widget

- A scroll bar appears on USB widget without need

- K4 - Volume sound when reached max doesn’t give beep alert sound and doesn’t flash in red,same with when volume reaches min

### CODE REVIEW

**Comment by javihernandez** [comment]
ok to test

**Comment by javihernandez** [comment]
> CI job failed: https://ci.gpii.net/job/gpii-app-tests/769/
```
Progress: 90%There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr is shown below.
```

ok to test

File: src/renderer/qssWidget/js/qssVolumeWidget.js [hunk 1]
```diff
@@ -444,19 +446,24 @@
      * subtracted from or added to the setting's value.
      * @return {Boolean} Whether there was a change in the setting's value.
      */
-    gpii.qssWidget.volumeStepper.makeRestrictedStep = function (that, value, schema, stepMultiplier) {
-        var step = schema.divisibleBy * stepMultiplier;
+    gpii.qssWidget.volumeStepper.makeRestrictedStep = function (that, value, schema, stepMultiplier, previousValue) {
+        var restrictedValue;
+        if (value === 0 && previousValue) {
```
Comment (amb26) [inline src/renderer/qssWidget/js/qssVolumeWidget.js]: Be clearer here about whether the expectation is that previousValue is being checked for being !== undefined or 0 (falsy), or just !== undefined

File: src/renderer/qssWidget/js/qssVolumeWidget.js [hunk 2]
```diff
@@ -444,19 +446,24 @@
      * subtracted from or added to the setting's value.
      * @return {Boolean} Whether there was a change in the setting's value.
      */
-    gpii.qssWidget.volumeStepper.makeRestrictedStep = function (that, value, schema, stepMultiplier) {
-        var step = schema.divisibleBy * stepMultiplier;
+    gpii.qssWidget.volumeStepper.makeRestrictedStep = function (that, value, schema, stepMultiplier, previousValue) {
```
Comment (amb26) [inline src/renderer/qssWidget/js/qssVolumeWidget.js]: Update JSDocs for this change in signature - I am assuming that this pull failed linting

File: messageBundles/gpii-app-qss-settings_en.json [hunk 3]
```diff
@@ -59,7 +59,7 @@
         "tooltip": "<p>Changes text size.</p><p>Works with many applications including Edge, Chrome, Firefox, Microsoft Office, and more.</p>",
         "title": "Zoom Top Window",
         "tip": "<p>Zoom Top Window</p><p>Note: Not all windows zoom.</p>",
-        "footerTip": "You can also use the “CTRL” key, plus SCROLL (wheel or gesture) to do Zoom Top Window without using Morphic."
+        "footerTip": "<a href=\"https://www.google.com/url?q=https://docs.google.com/document/d/1vnDDMvQENv3psIvZt_ibtedp4CWm62QnATcjYSxxN00/edit?usp%3Dsharing&sa=D&ust=1562689098659000&usg=AFQjCNE1ldXLzAvn2c41Hsiv5kyYzHyZQw\">Want a faster way to zoom a window?</a>"
```
Comment (javihernandez) [inline messageBundles/gpii-app-qss-settings_en.json]: This is a weird reference. Why not just adding this link? https://docs.google.com/document/d/1vnDDMvQENv3psIvZt_ibtedp4CWm62QnATcjYSxxN00

Also, is this supposed to end up in the morphic.world website?

Comment (krisYanachkov) [reply messageBundles/gpii-app-qss-settings_en.json]: Thanks for the observation.
Not sure, I copied it from the specification.

Comment (amb26) [reply messageBundles/gpii-app-qss-settings_en.json]: For me this link requires a sign-in

Comment (javihernandez) [reply messageBundles/gpii-app-qss-settings_en.json]: > For me this link requires a sign-in

That too, that's why I asked if this is supposed to end up in the morphic.world website.
Let's don't merge this until we get a public link for this.

**Pull request merged**